packages/liquidator/index.js

```diff
@@ -117,12 +122,6 @@ async function run({
       withdrawLiveness
     };
 
-    // Price feed must use same # of decimals as collateral currency.
-    let customPricefeedConfig = {
-      ...priceFeedConfig,
-      decimals: collateralCurrencyDecimals
```

Comment:
This previously worked by scaling the price feed by the number of decimals within the collateral currency. Now, the price feed has it's own scaling along with the collateral and decimals.